lib/authorisation_controller.js

```diff
@@ -1,5 +1,13 @@
 "use strict";
 
+/**
+ * Provides expressjs routing for authorisation endpoints.
+ * @module lib/authorisation_controller
+ * @copyright 2017 Genome Research Ltd
+ * @license GPL-3.0; see license in LICENSE
```

Comment:
Don;t think we are including a license annotation in every source file in other projects. So probably better if we just keep the annotation for copyright but remove the annotation for license.